lms/djangoapps/shoppingcart/models.py

```diff
@@ -1939,14 +1940,24 @@ def additional_instruction_text(self):
                 "If you haven't verified your identity yet, please start the verification process ({verification_url})."
             ).format(verification_url=verification_url)
 
-        refund_reminder = _(
-            "You have up to two weeks into the course to unenroll and receive a full refund."
-            "To receive your refund, contact {billing_email}. "
-            "Please include your order number in your email. "
-            "Please do NOT include your credit card information."
-        ).format(
-            billing_email=settings.PAYMENT_SUPPORT_EMAIL
-        )
+        if is_professional_mode_verified:
+            refund_reminder = _(
+                "You have only until two days after the course starts to unenroll and receive a full refund. "
+                "To receive your refund, contact {billing_email}. "
```

Comment:
Can we combine last 3 line as they are same for both `professional` and `verified` modes?